test/bigtest/tests/package-content-test.js

```diff
@@ -0,0 +1,110 @@
+import { beforeEach, describe, it } from '@bigtest/mocha';
+import { faker } from '@bigtest/mirage';
+
+import { expect } from 'chai';
+
+import setupApplication from '../helpers/setup-application';
+import PackageContentInteractor from '../interactors/package-content';
+
+const PKG = {
+  name: 'access_start_access_end_tests Package'
+};
+
+const testData = [
+  { pti: { titleInstance: { id: '1234', name: 'testInstance1' } }, accessStart: () => faker.date.recent().toISOString() },
```

Comment:
A bit clearer naming here will make maintaining these tests easier. We can do better than `testInstance#`